besu/src/main/java/org/hyperledger/besu/BesuInfo.java

```diff
@@ -27,6 +29,21 @@ public final class BesuInfo {
   private static final String VERSION = BesuInfo.class.getPackage().getImplementationVersion();
   private static final String OS = PlatformDetector.getOS();
   private static final String VM = PlatformDetector.getVM();
+  private static final String COMMIT;
+
+  static {
+    if (VERSION == null) {
+      COMMIT = null;
+    } else {
+      Pattern pattern = Pattern.compile("v?\\d*\\.\\d*\\.\\d*-\\w+-(?<commit>[0-9a-fA-F]{8})");
+      Matcher matcher = pattern.matcher(VERSION);
```

Comment:
Only the develop builds include the commit hash in the version at the moment, so this won't work for github release versions.

I tested locally using `./gradlew -Prelease.releaseVersion=99.9.0 -Pversion=99.9.0 assemble installDist` and the commit field in the response was null.

Reply:
Yeah, I'm not sure how else to get the commit hash. The purpose of the field is to give a specific commit at which the application was run, and for github releases there seems to be an accompanying tag, so the version itself can be used to find the exact commit if desired.

Do you think it would be worthwhile finding some other way to get the commit field populated?